src/pages/[platform]/build-a-backend/add-aws-services/analytics/sdk/index.mdx

```diff
@@ -0,0 +1,107 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Use AWS SDK',
+  description: 'For advanced use cases where Amplify does not provide the functionality, you can retrieve the escape hatch to access the AWSPinpoint instance.',
+  platforms: [
+    'swift',
+    'android'
+  ],
+  canonicalObjects: [
+    {
+      platforms: [
+        'android',
+      ],
+      canonicalPath: '/[platform]/build-a-backend/add-aws-services/analytics/sdk/'
+    }
+  ]
```

Comment:
It looks like these pages for swift and android aren't duplicates and so we don't need to use canonical objects here

```suggestion
  ]
```